test/common/common.go

```diff
@@ -211,6 +213,29 @@ func RegisterBrokerInSM(brokerJSON Object, SM *httpexpect.Expect, headers map[st
 		WithJSON(brokerJSON).Expect().Status(http.StatusCreated).JSON().Object().Raw()
 }
 
+func RegisterVisibilityForPlanAndPlatform(SM *httpexpect.Expect, planID, platformID string) {
+	SM.POST(web.VisibilitiesURL).WithJSON(Object{
+		"service_plan_id": planID,
+		"platform_id":     platformID,
+	}).Expect().Status(http.StatusCreated)
+}
+
+func RegisterVisibilityForBrokerID(SM *httpexpect.Expect, brokerID string) {
```

Comment:
What this function really does is essentially "publicize" all plans for a broker with the given ID by creating visibilities for each one. Why don't you rename the function to something more descriptive like "_PublicizeAllBrokerPlans_" or something similar?

Reply:
It is more like: `CreateVisibilitiesForAllBrokerPlans`. What do you think?

Reply:
Yeah, sounds good